libcef/browser/path_util_impl.cc

```diff
@@ -51,3 +51,38 @@ bool CefGetPath(PathKey key, CefString& path) {
 
   return false;
 }
+
+bool CefOverridePath(PathKey key, const CefString& path) {
+  int pref_key = base::PATH_START;
+  switch(key) {
+    case PK_DIR_EXE:
+      pref_key = base::DIR_EXE;
+      break;
+    case PK_DIR_MODULE:
+      pref_key = base::DIR_MODULE;
+      break;
+    case PK_DIR_TEMP:
+      pref_key = base::DIR_TEMP;
+      break;
+    case PK_FILE_EXE:
+      pref_key = base::FILE_EXE;
+      break;
+    case PK_FILE_MODULE:
+      pref_key = base::FILE_MODULE;
+      break;
+#if defined(OS_WIN)
+    case PK_LOCAL_APP_DATA:
+      pref_key = base::DIR_LOCAL_APP_DATA;
+      break;
+#endif
+    case PK_USER_DATA:
+      pref_key = chrome::DIR_USER_DATA;
```

Comment:
We currently allow customization of DIR_USER_DATA via CefSettings.user_data_path (in CefMainDelegate::PreSandboxStartup). If we add this method then we should delete CefSettings.user_data_path.